Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4614,6 +4636,9 @@ def to_context(self):
             if self.certificates:
                 domain_context['Certificates'] = self.create_context_table(self.certificates)
 
+            if self.rank:
+                domain_context['rank'] = self.create_context_table(self.rank)
```

Comment:
So in that case well have context path ```rank.rank```?

Reply:
no:
rank = [{"source": "x", "rank": "y"}, ...]

But it should be capitalized

Reply:
That's how grids work in CSP (you can see ExternalReference, Hash, DNSRecord and so on).